modules/system-view/src/main/java/org/apache/ignite/internal/systemview/SystemViewManagerImpl.java

```diff
@@ -188,6 +195,20 @@ public CompletableFuture<Void> completeRegistration() {
         return viewsRegistrationFuture;
     }
 
+    private void registerView(SystemView<?> view) {
+        if (views.containsKey(view.name())) {
```

Comment:
Am I right that we can't register two views with the same name but related to different schemas?

Reply:
View Definition API has no schema in it, thus you can't register views in different schema